seeds/users.js

```diff
@@ -0,0 +1,18 @@
+const tableName = require('../models/user')
+
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex('table_name').del()
```

Comment:
You seem to be passing in the string `table_name` not the variable

```suggestion
  return knex(tableName).del()
```